video_sampler/integrations/llava_chat.py

```diff
@@ -125,17 +126,16 @@ def summarise_image(self, image: Image):
             "stop": ["</s>", "Llama:", "User:"],
             "prompt": self.get_prompt(),
         }
-
         response = self.session.post(
-            f"{self.url}/completion",
+            f"{self.url}/completions",
```

Comment:
I tested this a while back so the api might've changed. Does llama file server now support openAI style for the image queries?

Reply:
Ah yes, they're installed through optional dependencies:
https://github.com/LemurPwned/video-sampler/blob/8f7b54f09ec923003b5dfc77c9ddce5b65faf423/pyproject.toml#L53
for example: 
```bash
python3 -m pip install video-sampler[language,clip,dev]
```

I'll add this to Readme and also [all] dependency -- thanks for pointing this out!

Reply:
Thanks, that's good to know. 

I tested against llama file convention which uses a different syntax for llava (which is an exception because for chat styles it understands openai-style). I think LM studio uses OpenAI syntax too (https://lmstudio.ai/docs/local-server). How about we extend the interface to support both? 

Would you be open to making that change? Basically, I was thinking that the user may specify the interface with some flag e.g. `--chat-mode=[openai,llamafile...]` and then the interface would morph adequately. 

Let me know if you'd like to make that change -- I can pick it up from there too.